### DISCUSSION TITLE
bookwyrm also for music albums/CDs ??

### DISCUSSION THREAD

**Discussion by rocky-III** [bookwyrm-social/bookwyrm#1594] [Ideas]
it would be really interesting to have something link bookwyrm also for  music albums/CDs

Could you imagine to integrate this into the software? 

Instead of  https://openlibrary.org/ one could use e.g. https://musicbrainz.org/ to get the basic infos about music albums/CDs

**Comment by mouse-reeve** [comment]
I totally get the appeal here! Unfortunately from a logistical standpoint, this isn't on my roadmap. I'm trying to keep bookwyrm's scope in check, and managing the complexity of book data is already so much a challenge that adding in other media is an overwhelming challenge for the size and development energy of this project. I'm going to convert this to a discussion since it isn't something I have the capacity to act on, but I appreciate the interest in expanding the conceptual model of managing media metadata and social activity around it to other things besides books.

**Comment by rocky-III** [comment]
Hi @mouse-reeve 
I understand what very well you are saying... but maybe with  music albums/CDs the project gets even more interests from some people and new developers  

I´m not a dev but from my understanding the basic work is already done by you... so it is about the api from e.g. musicbrainz and new classifications and labels in the bookwyrm db ... well it is for sure more than that...

How many devs besides of you are working right now on the project ?

**Reply by mouse-reeve** [reply]
I'm the only maintainer, there are usually one or two other people making contributions at any given moment. Unfortunately, managing data is a more complex problem than it appears on the surface. I've been keeping bookwyrm focused in scope both to try to keep it manageable as a hobby project, but also because, philosophically, I'm more interested in getting niche into books and reading, and serving the specificity of that rather than the general case. I think adding a variety of media types would confuse the use cases, and ultimately be a very different application than the one I'm trying to create. And that isn't necessarily a bad thing! But it's not a challenge that I'm up for, and I don't want to get anyone's hopes up that this project will transform into a general-purpose way to post about arbitrary types of media.
